src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -179,10 +179,10 @@
       (throw (IllegalArgumentException. "Rule query-params must be a map.")))
     (doseq [param (keys query-params)
             :let [value (get query-params param)]]
-      (when-not (string? param)
+      (when-not (keyword? param)
         (throw (IllegalArgumentException.
                 (str "The query-param '" param "' in the rule specified as "
-                     (pprint-rule rule) " is invalid. It should be a string."))))
+                     (pprint-rule rule) " is invalid. It should be a keyword."))))
```

Comment:
@camlow325 Wondering if I should delete this test...or change the invalid entry to be something other than a keyword or string, just so it doesn't look backwards and wrong